lib/libyear_bundler/models/gem.rb

```diff
@@ -59,10 +68,11 @@ def installed_version_sequence_index
       end
 
       def libyears
-        ::LibyearBundler::Calculators::Libyear.calculate(
-          installed_version_release_date,
-          newest_version_release_date
-        )
+        @libyears ||=
+          ::LibyearBundler::Calculators::Libyear.calculate(
+            installed_version_release_date,
+            newest_version_release_date
+          )
```

Comment:
Why add memoization here?

Reply:
Do not remember the reason. Retested without it - same result, so just removed that.